dll/win32/shell32/CDefView.cpp

```diff
@@ -60,8 +59,18 @@ enum  {
 
 typedef struct
 {
-    BOOL    bIsAscending;
+    INT8    Direction;
+    bool    bLoadedFromViewState;
+    bool    bColumnIsFolderColumn;
+    UINT8   Reserved; // Unused
     INT     ListColumn;
+
+    enum { UNSPECIFIEDCOLUMN = -1 };
+    void Reset()
+    {
+        *(UINT*)this = 0;
```

Comment:
Which fields above this will set to zero?

Reply:
All of them except ListColumn.